service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ReviewCMOControllerAboutToSubmitTest.java

```diff
@@ -145,6 +146,7 @@ void shouldNotModifyDataIfNoDraftCMOsReadyForApproval() {
         CaseData caseData = CaseData.builder()
             .draftUploadedCMOs(List.of())
             .uploadCMOEventData(UploadCMOEventData.builder().build()) // required due to the json unwrapping
+            .messageJudgeEventData(MessageJudgeEventData.builder().build()) // required due to the json unwrapping
```

Comment:
If you change the field in CaseData to
```java
    @Builder.Default
    private final MessageJudgeEventData messageJudgeEventData = MessageJudgeEventData.builder().build();
```
then you shouldn't need this.
Same could be done for the `UploadCMOEventData` if you want